src/hooks/useDoubleTokenConfig.ts

```diff
@@ -0,0 +1,21 @@
+import { Address } from "wagmi"
+
+import { useActiveChainConfig } from "@/hooks/useActiveChainConfig"
+
+import { fraxBpTokenAddress } from "@/constant/addresses"
+
+export const useDoubleTokenConfig = () => {
```

Comment:
I wonder if it would be better to just hardcode this onto the chain config itself?

Reply:
Not sure, this config is not driven by the chainId